src/utils.ts

```diff
@@ -682,19 +682,26 @@ export function reportUrlType(url: string, telemetryReporter: Readonly<Telemetry
 
 export async function reportFileExtensionTypes(telemetryReporter: Readonly<TelemetryReporter>): Promise<void> {
     const files = await vscode.workspace.findFiles('**/*.*', '**/node_modules/**');
-    const extensionMap: Map<string, number> = new Map<string, number>();
+    const extensionMap: Map<string, number> = new Map<string, number>([
+        ['html', 0],
+        ['css', 0],
+        ['js', 0],
+        ['ts', 0],
+        ['jsx', 0],
+        ['scss', 0],
+        ['json', 0],
+        ['mjs', 0],
```

Comment:
Nit: should we add an 'other' here? I guess we could calculate that info from totalsize - other files. But, it might make it easier on us if we threw everything else into 'other'.

No worries if it doesn't make sense. Only if we thing that information would be useful

Reply:
I'll add the `other` for the sake of clarity and so that the file list is self-contained.